languages/ruby/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,42 @@
+Ruby is a dynamic and strong typed language. On dynamic languages the type of a variable/object is resolved at runtime, which means that its value/type can be changed until the very last minute (when it gets parsed by the interpreter).
```

Comment:
```suggestion
Ruby is a dynamic and strongly typed language. In dynamic languages the type of a variable or object is resolved at runtime, which means that its value or type can be changed up to the very last minute (when it gets parsed by the interpreter).
```
My suggestion here is based solely on grammatical concerns. I'm not sure about the latter part of this sentence even with these changes. Can someone with more Ruby nous comment?

Reply:
"the very last moment" rather than "minute".  Ruby may be considered slow, but not quite that slow.

"For dynamic languages..."

Instead of "at runtime" it might be "during run-time" because the type referenced by a variable may change many times during run-time.

Reply:
> Ruby may be considered slow, but not quite that slow. 🤣 Good point